### DISCUSSION TITLE
Einheit wird einmal angezeigt / nach dem Reload verschwunden

### DISCUSSION THREAD

**Discussion by tschortschde** [Zefau/ioBroker.jarvis#2408] [Q&A] [answered]
Hallo,
ich habe ein seltsames Verhalten festgestellt.
Die Einheit wird nach dem Speichern des Backends im Slider angezeigt 
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/5ebf499e-9186-42dc-9e7c-7eb35e441581)
und wenn dann ein Reload gemacht wird ist die Einheit wieder weg.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/ea539fc9-14ca-480c-af12-0cac764d2cf6)

Woran liegt das?
Viele Grüße

**Comment by mcuiobroker** [comment]
Welche Version? Einträge im LevelBody? Fehlermeldungen im Browser-Log(F12)?

**Comment by tschortschde** [comment]
> Welche Version? 

(v3.1.9-beta.2)

> Einträge im LevelBody? 

![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/3d3e15aa-b2ed-40fd-ba3b-e83d27f69659)

> Fehlermeldungen im Browser-Log(F12)?

![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/23d75489-bd83-4543-ac3f-4fa1e8156494)

Wo/Wie kann ich den Fehler beheben?
VG

**Reply by mcuiobroker** [reply]
Mal auf das graue Dreieck klicken

**Comment by tschortschde** [comment]
Hast Du eine Idee?

**Reply by mcuiobroker** [reply]
Schon ein Upload gemacht?
iob upload jarvis
Nicht hier für die Performance. 
Hier Einstellungen vom Gerät

**Comment by tschortschde** [comment]
Was meinst Du mit Upload. Update ?
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/8d2f92a7-b2fc-4ee3-86a6-ab4086d09593)

**Comment by mcuiobroker** [comment]
Zeig bitte erstmal die Einstellungen zum Gerät.
Bitte auch als Text.

**Comment by tschortschde** [comment]
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/2aa982cf-0a2a-45ba-aeaa-2a56410ce399)
Datenpunkt (Anzeige):
knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Status_aktuelle_Position
Datenpunkt (Trigger):
knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_absolut
Icon (JSON Format):
{"{knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Status_aktuelle_Position}>50 && {knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Verfahrstatus}=true":"window-shutter blink","{knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Status_aktuelle_Position}<=50 && {knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Verfahrstatus}=true":"window-shutter-open blink","{knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Status_aktuelle_Position}>50 && {knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Verfahrstatus}=false":"window-shutter","{knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Status_aktuelle_Position}<=50 && {knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost_-_Verfahrstatus}=false":"window-shutter-open","default":"window-shutter"}
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/fa313ff2-bdf8-4f1a-a36c-46a176cb25ad)
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/e1e04f8c-0562-41ce-b15a-20c650c224ff)
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/5791f7a7-dbce-4572-9ba3-49559e166d05)

**Comment by mcuiobroker** [comment]
Für das Icon
```
{"{val}>50 && {knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost-Verfahrstatus}=true":"window-shutter blink","{val}<=50 && {knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost-Verfahrstatus}=true":"window-shutter-open blink","{val}>50 && {knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost-Verfahrstatus}=false":"window-shutter","{val}<=50 && {knx.0.001_Büro.3_Beschattung_Fenster.E01_Büro_Ost-_Verfahrstatus}=false":"window-shutter-open","default":"window-shutter"}
```
Wer hat den `level` auf `level1` geändert? DAS MUSS `level` sein!
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/f11a176a-06a9-47d8-9258-e1e2e8ce9e93)

Dann bitte die Einstellungen von LevelBody und BlindLevelAction
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/f91ce93d-ad6c-40f9-8851-a7ccd9b02de0)

Und einmal auf das Dreieick klicken
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/60b74d4d-12b5-4a8c-9f3d-53a421b38297)

Beschattungsautomatik
Datenpunkt Eigenschaften
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/94b91eb2-a9a8-4334-9fe0-9a82117a3100)
Kann nicht funktionieren, Warum?

**Comment by tschortschde** [comment]
Die Anpassungen habe ich vorgenommen.
Nun noch die weiteren Punkte...

>Dann bitte die Einstellungen von LevelBody und BlindLevelAction

![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/242f8859-66bd-48ed-98a9-c1e1909a9146)
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/20defa29-6eed-47b5-8f56-6c8bd472c5bd)

Diesen Fehler habe ich nicht mehr:

>Und einmal auf das Dreieick klicken
>![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/21a04797-a6da-4b84-bb54-aa37c457cfc6)

>Beschattungsautomatik
>Datenpunkt Eigenschaften

Habe ich entfernt

**Comment by mcuiobroker** [comment]
Wie sieht es aus? Veränderung?

**Comment by tschortschde** [comment]
Beim ersten Speichern: 
% vorhanden
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/d4b737f1-cf75-42db-b747-1c60eda965b0)

Beim Aktualisieren:
% weg 
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/9ec2898a-2258-4268-94d7-15ea42715293)

**Comment by mcuiobroker** [comment]
Dann versuchen wir es mit dem Upload
iobroker console
```
iob upload jarvis
```

**Comment by tschortschde** [comment]
Upload hat funktioniert 
Verhalten bleibt jedoch gleich ;-(

**Comment by mcuiobroker** [comment]
Dann gibt es nur noch die Möglichkeiten

- Web-Instanz stoppen und starten
- Browser Cache löschen

mehr fällt mir dann nicht ein.
Doch sind im Layout Einstellungen anders als im Gerät?

Performance denn jetzt besser?

**Comment by tschortschde** [comment, marked as answer]
Hab das Gerät im Layout erneut hinzugefügt und festgestellt, dass es dann funktioniert.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/d56a60e9-023a-4cdf-b5bc-47f6ebdc3a7a)
Der Unterschied liegt darin, dass der Level Body
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/1bf5ecb8-2bd9-488d-92d4-efd5c3deaaf2)
manuell erfasst wurde !?

**Comment by mcuiobroker** [comment]
Dann sind wir hiermit fertig?

**Reply by tschortschde** [reply]
Ja, vielen Dank für die Unterstützung
